.nx/cache/run.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Remove the entire `.nx` folder! You shouldn't commit this. Don't add it to .gitignore, leave it as is.